docs/zh_cn/inference/pipeline.md

```diff
@@ -116,6 +116,25 @@ for item in pipe.stream_infer(prompts, gen_config=gen_config):
     print(item)
 ```
 
+- **计算 logits & ppl:**
+
+```python
+from transformers import AutoTokenizer
+from lmdeploy import pipeline
+pipe = pipeline('internlm/internlm2-chat-7b')
+tokenizer = AutoTokenizer.from_pretrained('internlm/internlm2-chat-7b', trust_remote_code=True)
+
+# logits
+chat = [
+   {"role": "user", "content": "Hello, how are you?"},
+]
+input_ids = tokenizer.apply_chat_template(chat)
+logits = pipe.get_logits(input_ids)
+
+# ppl
+ppl = pipe.get_ppl(input_ids)
```

Comment:
Just an interesting result: ppl is different (around 4%) between pytorch and turbomind for this example
|| Turbomind | PyTorch |
|:-|:-:|:-:|
| ppl| 5.5916224 | 5.3524413|

Reply:
It's possible. They use different cuda kernels